helpers/slots.js

```diff
@@ -161,4 +161,24 @@ module.exports = {
 	calcRound(height) {
 		return Math.ceil(height / this.delegates);
 	},
+
+	/**
+	 * Calculate starting height of the round
+	 *
+	 * @param round
+	 * @return {number}
+	 */
+	calcRoundStartHeight(round) {
+		return (round < 1 ? 0 : round - 1) * this.delegates + 1;
```

Comment:
Why such a condition? Round cannot be `0`, same for height.

Reply:
That's why its protecting that part, if someone passed that round as 0 or negative.

Reply:
Ok, it's fine.